proxysql-scheduler/local/build-binary-proxysql-scheduler

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+
+# ------------------------------------------------------------------------------
+# Declare all input variables
+# ------------------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------------------
+# set working dir
+# ------------------------------------------------------------------------------
+#mkdir -p ${1:-./build}
+WORKDIR=$(cd ${1:-./build}; pwd -P)
+INSTALL_DIR=${WORKDIR}/DESTDIR
+SOURCEDIR=$(cd ${2:-$(dirname $0)/../sources/proxysql-scheduler}; pwd -P)
+
+pushd /tmp
+    #sudo wget -q https://golang.org/dl/go1.15.8.linux-amd64.tar.gz
+    #sudo tar -C /usr/local -xzf go1.15.8.linux-amd64.tar.gz
+    #export PATH=$PATH:/usr/local/go/bin
+    #sudo /usr/local/go/bin/go version
+    #sudo rm go1.15.8.linux-amd64.tar.gz
+    sudo apt update
+    sudo apt install -y golang
+popd
+
+pushd ${WORKDIR}
+    echo "Getting dependencies"
+    while read l; do
+        if [[ $l == github* || $l == golang* ]]; then
+            sudo go get -v "$l"
+        fi
+    done < <(sudo go list -f '{{ join .Imports "\n" }}')
+
+    # todo: fix these dependencies to be able to get them automatically
+    sudo go get -v github.com/Tusamarco/toml
+    sudo go get -v golang.org/x/text/language
+    sudo go get -v golang.org/x/text/message
+
+    sudo go build
```

Comment:
First of all we need to setup go env correctly. yes if we use from package it would be ok but centos systems doesn't have latest go-lang in repo and there might be an issue here
Also maybe it would be better to rework this and use go dep (which is currently deprecated ) or migrate to go modules

Reply:
As above. We do not want Centos at the moment. Only Ubuntu Focal.